src/tap.cpp

```diff
@@ -71,8 +186,8 @@ void TAP::fdActivated() {
     if (bytesRead > 0) {
```

Comment:
In case the watch needs to send something to the companion before the companion sent anything, our MTU isn't set correctly yet (defaults to 244) and we would send an invalid frame to the companion... I feel like we should refuse sending messages as long as the MTU hasn't been retrieved.

I'd be surprised if there's no DBus way to retrieve the MTU of the BLE connection from BlueZ without waiting for a Read or Write first. Maybe with `AcquireNotify` and `AcquireWrite` ? If not, I don't know if this is a BlueZ limitation or a limit of BLE but either way, we could document that companions need to initiate a dummy Read() first, so the MTU propagates to the TAP interface and we don't send invalid messages.